src/datagrid/DataGrid.tsx

```diff
@@ -275,6 +275,8 @@ export const DEFAULT_COLUMN_WIDTH: number = 100;
 export const DEFAULT_CURRENT_PAGE_NUMBER: number = 1;
 export const DEFAULT_PAGE_SIZE: number = 10;
 export const DEFAULT_TOTAL_ITEMS: number = 0;
+export const DEFAULT_PAGE_SIZES: string[] = ["10", "20", "50", "100", "Custom"];
+export const DEFAULT_CUSTOM_PAGE_SIZE_LIMIT: number = 1000;
```

Comment:
```suggestion
export const MAX_PAGE_SIZE: number = 1000;
```
Please add prop for MAX_PAGE_SIZE as well in Pagination prop. If user don't provide it then default value is 1000